WPFHospitalEditor/Building.xaml.cs

```diff
@@ -53,17 +100,18 @@ private void setFloorComboBox()
 
         private void back_Click(object sender, RoutedEventArgs e)
         {
-            clearAll();
+            clearAll();           
+            HospitalMap hospitalMap = new HospitalMap(mapObjectController.getAllMapObjects());
```

Comment:
Unnecessarily reload objects for hospitalmap.

Reply:
Firstly hospitalMap object was closed after clicking on building, but now is set to hide after clicking on building, so it is no longer reloading.